capa/main.py

```diff
@@ -429,40 +429,38 @@ def is_running_standalone() -> bool:
     return hasattr(sys, "frozen") and hasattr(sys, "_MEIPASS")
 
 
-def get_default_root() -> str:
+def get_default_root() -> Path:
     """
-    get the file system path to the default resources directory.
-    under PyInstaller, this comes from _MEIPASS.
-    under source, this is the root directory of the project.
+    Get the file system path to the default resources directory.
+    Under PyInstaller, this comes from _MEIPASS.
+    Under source, this is the root directory of the project.
     """
     if is_running_standalone():
-        # pylance/mypy don't like `sys._MEIPASS` because this isn't standard.
-        # its injected by pyinstaller.
-        # so we'll fetch this attribute dynamically.
-        return getattr(sys, "_MEIPASS")
-    else:
-        return os.path.join(os.path.dirname(__file__), "..")
+        try:
+            meipass = Path(getattr(sys, "_MEIPASS"))
+            return meipass
+        except AttributeError:
+            pass
+    # Return the root directory of the project when not running from a PyInstaller'd executable
+    return Path(__file__).resolve().parent.parent
 
 
-def get_default_signatures() -> List[str]:
+def get_default_signatures() -> List[Path]:
     """
-    compute a list of file system paths to the default FLIRT signatures.
+    Compute a list of file system paths to the default FLIRT signatures.
     """
-    sigs_path = os.path.join(get_default_root(), "sigs")
+    sigs_path = get_default_root() / "sigs"
     logger.debug("signatures path: %s", sigs_path)
 
     ret = []
-    for root, _, files in os.walk(sigs_path):
-        for file in files:
-            if not (file.endswith(".pat") or file.endswith(".pat.gz") or file.endswith(".sig")):
-                continue
-
-            ret.append(os.path.join(root, file))
+    for file in sigs_path.rglob("*"):
+        if file.is_file() and file.suffix.lower() in (".pat", ".pat.gz", ".sig"):
+            ret.append(file)
```

Comment:
i think `os.walk` is recursive while `rglob(*)` only inspects within the current directory. so, this is a minor change in behavior. mostly just mentioning it here, since i dont think it affects our code at all.